runtime/compiler/control/rossa.cpp

```diff
@@ -2102,11 +2102,6 @@ aboutToBootstrap(J9JavaVM * javaVM, J9JITConfig * jitConfig)
       return -1;
       }
 
-   if (!TR::Options::getCmdLineOptions()->allowRecompilation() || !TR::Options::getAOTCmdLineOptions()->allowRecompilation())
```

Comment:
I think that the reasoning behind this statement is chtable optimizations rely on recompilation to function.
I am concerned that eliminating this line, a run -Xjit:disableRecompilation may not run properly.